debian/po/pt_BR.po

```diff
@@ -2554,7 +2557,7 @@ msgstr ""
 
 #: ../../uaclient/messages/__init__.py:1463
 msgid "All Updates for the Robot Operating System"
-msgstr ""
+msgstr "Todas as Atualizações para o Robot Operating System"
```

Comment:
s/Atualizações/atualizações

Reply:
I kept the uppercase to match the uppercase in All Updates.
I think we must have a policy for this, because it is not consistent across messages